src/Device.cpp

```diff
@@ -42,6 +42,21 @@ uint32_t Device::getSerialNumber() const {
     return m_serial_number;
 }
 
+canbus::Message Device::querySetId(uint8_t id) {
+    canbus::Message message;
+    message.can_id = 0x600;
+    message.size = 8;
+    message.data[0] = 0x40;
```

Comment:
master writes to slave

```suggestion
    message.data[0] = 0x23;
```